src/workflows-app/SubmissionConfig.js

```diff
@@ -197,6 +199,14 @@ export const BaseSubmissionConfig = (
         message: 'You may check on the progress of workflow on this page anytime.',
         timeout: 5000,
       });
+      Ajax(signal).Metrics.captureEvent(Events.workflowsAppLaunchWorkflow, {
+        ...extractWorkspaceDetails(workspace),
+        methodName: method.name,
+        methodId,
+        methodVersionId: selectedMethodVersion.method_version_id,
+        methodSource: method.source,
+        previouslyRun: method.last_run.previously_run,
+      });
```

Comment:
Two notes here:
1. This shouldn't throw any errors even if the network call errors out, which means it won't redirect to the `catch` branch (maintaining the correct user flow)
2. Any submit events we want to test will need to mock out the Metrics.captureEvent function, ex:
https://github.com/DataBiosphere/terra-ui/blob/e33a1b040bf7f84c0b9a4ed6f39e1307d34ec5f6/src/analysis/Analyses.test.ts#L102
https://github.com/DataBiosphere/terra-ui/blob/e33a1b040bf7f84c0b9a4ed6f39e1307d34ec5f6/src/analysis/Analyses.test.ts#L114-L116